.vscode/settings.json

```diff
@@ -1,13 +1,11 @@
 {
-    "python.linting.pylintEnabled": false,
-    "python.linting.flake8Enabled": true,
-    "python.linting.enabled": true,
```

Comment:
thought: I think there's another discussion that could be had on whether we want to encourage ehrQL code to adhere to a particular style. The tooling options — and things like automated formatting — that we provide may have an influence on that. 

But I think it's fine, at this time, to ditch these options to suppress unwanted notifications.